### DISCUSSION TITLE
detailsRow Styling

### DISCUSSION THREAD

**Discussion by samuelthomps0n** [Laravel-Backpack/community-forum#754] [Q&A (Help)]
I'm using the detailsRow to add extra data to the table and this all works fine, however the detailsRow template is rending only within the width of the first column and not stretching across the whole table. How can I fix this? 

![Screenshot 2023-11-25 at 10 23 49](https://github.com/Laravel-Backpack/community-forum/assets/3295986/4129a0fb-6f93-435a-bdd2-7a72aa5dc014)

**Comment by pxpm** [comment]
Hello @samuelthomps0n 

 was not able to reproduce the issue you mentioned. 

I am guessing it could be something related with the HTML you added inside the list template, or some custom css/html you added somewhere in the page. 

I just enabled details row in a crud with: `$this->crud->enableDetailsRow()` and by default it stretches to the whole content:
![image](https://github.com/Laravel-Backpack/community-forum/assets/7188159/265f779f-4325-471d-b28c-0cef098acc9f)

Mind checking the generated html and post it here ? 

Cheers

**Comment by jcastroa87** [comment]
We haven’t heard back from you in a while, so we’re closing this issue for now. If you still need assistance, please don’t hesitate to reopen the issue or create a new one.

Thank you, and take care!

Cheers.
